adoc/Private-Registry-Administration.adoc

```diff
@@ -743,3 +743,206 @@ suse-registry-harbor-registry-65854df7bc-mrfnj     2/2     Running   0
+trivy:
+  podAnnotations:
+    backup.velero.io/backup-volumes: data
+database:
+  podAnnotations:
+    backup.velero.io/backup-volumes: database-data
+redis:
+  podAnnotations:
+    backup.velero.io/backup-volumes: data
+----
++
+[source,bash]
+----
+helm -n registry upgrade suse-registry ./harbor -f harbor-values.yaml
+----
++
+.External Services
+[NOTE]
+====
+If using an external service, such as a managed PostgreSQL database, adding the `database` entry is not required. The same applies for redis when using an external redis and for registry when using an external storage backend for storing images and artifacts.
+====
+
+. Perform the backup by running the `velero backup create <backup-name> --include-namespaces <namespace-to-backup>` command, for example:
++
+[source,bash]
+----
+> velero backup create registry-backup --include-namespaces registry
+Backup request "registry-backup" submitted successfully.
+Run `velero backup describe registry-backup` or `velero backup logs registry-backup` for more details.
+----
++
+.Scope of the backup
+[NOTE]
+====
+Velero supports performing the backup of all Kubernetes cluster resources, however in this case the backup is performed for the registry namespace only. In some cases, such as when using `cert-manager` or `redis-operator`, you might need to include those namespaces on the backup.
+====
++
+
+. Run `velero backup describe <backup-name>` to check the backup status and ensure its completion. For example:
++
+[source,bash]
+----
+> velero backup describe registry-backup
+Name:         registry-backup
+Namespace:    velero
+Labels:       velero.io/storage-location=default
+Annotations:  <none>
+
+Phase:  Completed
+
+Namespaces:
+  Included:  registry
+  Excluded:  <none>
+
+Resources:
+  Included:        *
+  Excluded:        <none>
+  Cluster-scoped:  auto
+
+Label selector:  <none>
+
+Storage Location:  default
+
+Snapshot PVs:  auto
+
+TTL:  720h0m0s
+
+Hooks:  <none>
+
+Backup Format Version:  1
+
+Started:    2020-12-17 12:00:12 +0100 CET
+Completed:  2020-12-17 12:01:36 +0100 CET
+
+Expiration:  2021-01-16 12:00:12 +0100 CET
+
+Persistent Volumes: <none included>
+
+Restic Backups (specify --details for more information):
+  Completed:  4
+----
+
+.Backup options
+[NOTE]
+====
+For more advanced options, such as scheduling backups and configuring its expiration time, check the link:https://documentation.suse.com/suse-caasp/4.2/html/caasp-admin/_backup_and_restore_with_velero.html[Velero documentation].
+====
+
+===== Restore
+
+. If the restoration is performed on a new Kubernetes cluster, install Velero by following the link:https://documentation.suse.com/suse-caasp/4.2/html/caasp-admin/_backup_and_restore_with_velero.html[documentation]. Make sure to use the same storage backend used to perform the backup so Velero can access the backups performed previously. If restoring on the same cluster, make sure to completely delete the existing {spr} deployment, including its namespace.
```

Comment:
```suggestion
. If the restoration is performed on a new Kubernetes cluster, install Velero by following the link:https://documentation.suse.com/suse-caasp/4.2/html/caasp-admin/_backup_and_restore_with_velero.html[documentation]. Make sure to use the same storage backend used to perform the backup, so that Velero can access the backups performed previously. If restoring on the same cluster, make sure to completely delete the existing {spr} deployment, including its namespace.
```

Reply:
done